### ISSUE TITLE
Alternate color scheme for colorblind users

### ISSUE THREAD

**Issue by boosterbeetle** [mikebryant/ac-nh-turnip-prices#254]
Just noticed the new colors to represent good/bad prices but as someone with red/green colorblindness it's pretty difficult to distinguish between the shades of green. Also checked out the dark mode equivalent and the shades of purple? are also difficult to tell. Not a deal breaker by any means but just leaving some feedback in case you want to add support for other schemes in the future. All in all, great work.

**Comment by theRTC204** [comment]
Great suggestion. There are a handful of things we should do for accessibility.

**Comment by davidcorry** [comment]
I've made an initial pass at a higher contrast theme, that should (hopefully) work across the different forms of colorblindness. I don't have any colorblindness, though, so I was relying on an app to simulate it for me. Any feedback from folks who this would benefit would be great! (@boosterbeetle)

I haven't made a dark mode, but depending on interest I can see about how to handle the logic for that.

<img width="849" alt="image" src="https://user-images.githubusercontent.com/265951/82398317-7664fd80-9a07-11ea-94cd-f73dc2cfb993.png">
<img width="1235" alt="image" src="https://user-images.githubusercontent.com/265951/82398351-88df3700-9a07-11ea-8583-07f52b2a888e.png">

**Comment by boosterbeetle** [comment]
@davidcorry Screenshot looks good! Easier to tell between different sets of values now.

**Comment by davidcorry** [comment]
Rad! If anything needs adjusting, let me know. I’ll likely come up with a dark mode version in the next day or two and hopefully we can get it in soon.

**Comment by boosterbeetle** [comment]
Yeah, not sure if others want to pitch in but for me, I had a hard time distinguishing between the best and second best values in the previous color scheme (shades of green used were very similar). It's solved here though as one of the blue is much darker than the other one. Don't know if that still causes problems with other users though.
